packages/web-components/fast-foundation/src/tabs/tabs.ts

```diff
@@ -126,6 +128,11 @@ export class Tabs extends FASTElement {
         }
     }
 
+    private getDirection = (): Direction => {
+        const dirNode: HTMLElement | null = this.parentElement!.closest("[dir]");
```

Comment:
It looks like slider has similar logic to determine direction. Can you move this to a utility that both can leverage so that we can ensure we're determining direction consistently across components?

Reply:
Good call

Reply:
Moved getDirection to a utility.

@marjonlynch Also removed the `setThumbPositionForOrientation` inside of the original function from *Slider* as that is already run in connectedCallback for slider after the `getDirection` function runs.